dependencies.gradle

```diff
@@ -11,7 +11,7 @@ def VERSIONS = [
         'com.google.dagger:dagger-compiler:2.11',
         'com.google.guava:guava:latest.release',
         'com.google.inject:guice:4.1.0',
-        'com.hazelcast:hazelcast:4.+',
+        'com.hazelcast:hazelcast:3.+',
```

Comment:
I defaulted to 3 since users bumping up their Micrometer probably would not like it bumping IMDG version.

Reply:
Hmm, I think the other way, they want to upgrade Hazelcast to 4 unless they already have a dependency somewhere to Hazelcast 3. What are your thoughts? CC: @alparslanavci  @mesutcelik

Reply:
I'm fine either way